k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/config-template.yaml

```diff
@@ -44,22 +44,22 @@ cortx:
     auth_user: user_name
     auth_admin: sgiamadmin
     auth_secret: s3_auth_admin_secret
-    iam:
-      endpoints:
-      - https://<<.Values.cortx.io.svc>>:9443
+    iam:                                                               #Deleted
```

Comment:
rgw is new section hence nothing can be deleted. Mention a separate section named s3

```
s3:                                                       # deprecated. 
  iam:                                                    # deprecated. 
...
```

Reply:
added a s3 section and marked it as deprecated